doc/tags_old.txt

```diff
@@ -1070,10 +1070,10 @@ v2_3	Attempt to restart a crashed entry few times before shutting down the
  	KNOWN ISSUES: If using condor 7.4.0-7.4.2 and 7.5.0, USE_VOMS_ATTRIBUTESshould be set to false for Collector and negotiator to avoid potential problems and memory leaks in GSI libraries.
 
 v2_2	Bug Fix: GLEXEC_JOB and GLEXEC_STARTER were not published in glideins
-          classds. This was preventing psuedo interactive monitoring to work
+          classds. This was preventing pseudo interactive monitoring to work
```

Comment:
Should `classds` also be changed to `classads`? If yes, might need to be fixed manually.

Reply:
fixed